api/src/main/resources/application.properties

```diff
@@ -1,4 +1,4 @@
-spring.datasource.url=jdbc:postgresql://${AB2D_DB_HOST}:${AB2D_DB_PORT}/${AB2D_DB_DATABASE}
+spring.datasource.url=jdbc:postgresql://${AB2D_DB_HOST}:${AB2D_DB_PORT}/${AB2D_DB_DATABASE}?sslmode=${AB2D_DB_SSL_MODE}
```

Comment:
Can we specify the default value _inline_ as well, so that we don't even need to bother specifying the environment variable if we don't want SSL?  `${AB2D_DB_SSL_MODE:allow}` or similar.

Reply:
done